tests/test_evpn_tunnel.py

```diff
@@ -0,0 +1,690 @@
+
+      assert len(fvs) >= len(expected_attributes), "Incorrect attributes"
+
+      attr_keys = {entry[0] for entry in fvs}
+
+      num_match = 0
+      for name, value in fvs:
+          if name in expected_attributes:
+                 if expected_attributes[name] == value:
+                    num_match += 1
+      if num_match == len(expected_attributes):
+         retkey.append(key)
+
+    return retkey
+       
+      
+def create_evpn_nvo(dvs, nvoname, tnl_name):
+    conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+    attrs = [
+            ("source_vtep", tnl_name),
+    ]
+
+    # create the VXLAN tunnel Term entry in Config DB
+    create_entry_tbl(
+        conf_db,
+        "VXLAN_EVPN_NVO", '|', nvoname,
+        attrs,
+    )
+
+def remove_evpn_nvo(dvs, nvoname):
+    conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+    delete_entry_tbl(conf_db,"VXLAN_EVPN_NVO", nvoname,)
+
+def create_vxlan_tunnel(dvs, name, src_ip, dst_ip = '0.0.0.0', skip_dst_ip=True):
+    conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+    attrs = [
+            ("src_ip", src_ip),
+    ]
+
+    if not skip_dst_ip:
+        attrs.append(("dst_ip", dst_ip))
+
+    # create the VXLAN tunnel Term entry in Config DB
+    create_entry_tbl(
+        conf_db,
+        "VXLAN_TUNNEL", '|', name,
+        attrs,
+    )
+
+def create_vxlan_tunnel_map(dvs, tnl_name, map_name, vni_id, vlan_id):
+    conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+    attrs = [
+            ("vni", vni_id),
+            ("vlan", vlan_id),
+    ]
+
+    # create the VXLAN tunnel Term entry in Config DB
+    create_entry_tbl(
+        conf_db,
+        "VXLAN_TUNNEL_MAP", '|', "%s|%s" % (tnl_name, map_name),
+        attrs,
+    )
+
+def create_evpn_remote_vni(dvs, vlan_id, remote_vtep, vnid):
+    app_db = swsscommon.DBConnector(swsscommon.APPL_DB, dvs.redis_sock, 0)
+    create_entry_pst(
+        app_db,
+        "VXLAN_REMOTE_VNI_TABLE", ':', "%s:%s" % (vlan_id, remote_vtep),
+        [
+            ("vni", vnid),
+        ],
+    )
+    time.sleep(2)
+
+def remove_vxlan_tunnel(dvs, tnl_name):
+    conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+    # create the VXLAN tunnel Term entry in Config DB
+    delete_entry_tbl(
+        conf_db,
+        "VXLAN_TUNNEL", tnl_name,
+    )
+
+def remove_vxlan_tunnel_map(dvs, tnl_name, map_name,vni_id, vlan_id):
+    conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+    attrs = [
+            ("vni", vni_id),
+            ("vlan", vlan_id),
+    ]
+
+    # create the VXLAN tunnel Term entry in Config DB
+    delete_entry_tbl(
+        conf_db,
+        "VXLAN_TUNNEL_MAP", "%s|%s" % (tnl_name, map_name),
+    )
+
+def remove_evpn_remote_vni(dvs, vlan_id, remote_vtep ):
+    app_db = swsscommon.DBConnector(swsscommon.APPL_DB, dvs.redis_sock, 0)
```

Comment:
There are also connectors you can use in `dvs` like `get_app_db` and `get_asic_db`.